cpp/log/log_lookup_test.cc

```diff
@@ -59,6 +62,13 @@ class LogLookupTest : public ::testing::Test {
                   new MerkleVerifier(new Sha256Hasher())) {
     // Set some noddy STH so that we can call UpdateTree on the Tree Signer.
     store_.SetServingSTH(ct::SignedTreeHead());
+    // Force an empty sequence mapping file:
```

Comment:
Not directly related to this pull request, but I'm not entirely clear why this test needs a ```ConsistentStore``` (```LogLookup``` only needs a ```ReadOnlyDatabase```)? Is this testing a lot more than just ```LogLookup```?